### ISSUE TITLE
Items that are not showing

### ISSUE THREAD

**Issue by TeoTwawki** [TeoTwawki/LootBox#7]
Has to do with the type of the item and the flagging on it.

Mainly non equip and non usable are not showing atm.

Once we have all the items displaying properly again going to put out a new release exe.

**Comment by RavenTactics** [comment]
Currently the UI doesn't display quantities of items in the stack, and this may already be something that is planned. 

This is the list of items that are not listed in Lootbox but are in bags in-game.  To verify I loaded Lootbox first and then logged in game on 1 character.  I can check other characters at a later time. 

### Inventory 
- Crystals; Water
- Gigas Socks
- Gigas Necklace
- Goblin Mask
- Linkpearl
- **Ore**;  Copper, Tin, Mythril, Darksteel, Silver, Gold
- Flintstone
- Insect Wing
- Bird Feather
- Bone Chip
- Beetle Shell
- Mistletoe
- Dryad Root
- Black Ink
- Paprika Grass
- Remi Shell
- Makibishi
- Mizu-Deppo
- Shihei
- Shinobi-Tabi
- Beetle Arrowhead
- Fang Arrowhead
- Marble
- Helmet Mole
- Sanjaku-Tenugui

### Mogsafe (1)
- Adventurer Coupon
- Wild Rabbit Tail
- Smooth Stone
- Supplies Order
- White Jar
- Yutunga Sulfur
- Shaman Garlic
- Red Cryptex
- Citidal Tapestry
- Bonbori 
- Orcish Armor Plate
- Katana-kazari
- Festive Dolls
- Damselfly Worm
- Bahut
- Caisson
- Magicked Skull
- Crab Apron
- Bloody Robe
- Dangruf Stone
- Lufet Salt
- 3Leaf Mandra Bud
- Silver Engraving
- Yagudo Caulk
- Magicked Steel
- Squamous Hide
- Jade Cryptex

### Mogsafe (2)
- Marauder's Knife
- Water Jug

### Storage
- Scroll of Warp
- **Tarut: The**; Hermit, King
- Beastmen's Seal
- Dragon Blood 

### Mog Locker
n/a

### Mog Satchel
- Theif's Tools
- **Chest Keys**; Oztroja, Giddus, Ranperre, Onzozo
- **Coffer Keys**; Boyahda
- Brass Canteen
- Pickaxe
- Sickle 
- Lanolin
- Zeruhn Soot
- **Ore**;  Iron, Platinum
- **Beastcoin**; Gold, Mythril, Silver
- Spider Web
- Lumbering Horn
- **Flowers**; Winjuruit, Lilac, Marguerite
- Sand Bat Fang
- Kindred Seal
- Fresh Blood

### Mog Sack
- **Crystals**; Lightning
- Goblin mask
- Ahriman Lens
- Crying Mustard
- Slime Oil
- Sage
- Flint Stone
- Giant Bird Feather
- Bone Chip
- Beehive Chip
- Mercury
- Mistletoe
- Malbro Vine
- Ahriman Tears
- Dryad Root
- Fiend Blood
- Carbon Fiber
- Glass Fiber
- Rock Salt
- Animal Glue
- Revival Root
- Venom Dust
- Paralysis Dust
- Poison Flour
- Treant Blub
- Magic Pot Shards
- Sulfur
- Artificial Lens
- Wyvern Wing
- Doll Shard
- **Bolt Head**; Bloody, Acid
- Bugbear Mask
- Moblin Mask

### Mog Case
- Coral Butterfly
- Quadav Helm
- Goblin Mail
- Quadav Backplate
- **Ingot**; Bronze, Brass, Tin
- Iron Scales
- Iron Chain
- Log; Mahogany, Rosewood, Ebony
- Lumber; Rattan
- Colored Rocks; Yellow, White
- **Gems**; Light Opan, Amethyst, Tourmaline, Sadonyx, 
- Yagudo Feather
- Bird Feather
- Bat Wing
- Coeurl Wisker
- Dhalia
- Antican Pauldron
- Crawler Calculus
- Moblin Helm
- Moblin Mail
- Kopparnickle Ore
- Igneous Rock
- Hoary Bomb Ash
- Marble
- Flawed Garnet
- Aramid Fiber

### Mog Wardrobe (1)
- Bastokan Ring (Demiurge Custom Ring) 

### Mog Wardrobe (2)
- Archer's Knife (See image)
![image](https://user-images.githubusercontent.com/37428609/41425298-2454e916-6fb5-11e8-97f8-99471ae3a40d.png)


### Mog Wardrobe (3)
- Field Boots

### Mog Wardrobe (4)
- Harp

**Comment by TeoTwawki** [comment]
looks like the test build I shared with you last night actually made the problem *worse*

**Comment by RavenTactics** [comment]
Steps taken:  Deleted the EXE. Doanloaded and move the new EXE (https://github.com/TeoTwawki/LootBox/releases) into the same folder. 

This is an update to the list from the comments above. 

### Inventory 
Fixed: 
 - Water Crystals now display

New item: 
- linksell maybe? 
![image](https://user-images.githubusercontent.com/37428609/41427037-7f471880-6fba-11e8-988f-d6642e69bc22.png)


### Mogsafe (1) 
No change detected in new exe build. 

### Mogsafe (2)
No change detected in new exe build. 

### Storage
No change detected in new exe build. 

### Mog Locker
n/a

### Mog Satchel
No change detected in new exe build. 

### Mog Sack
Fixed: 
 - Lightning Crystals now display

### Mog Case
No change detected in new exe build. 

### Mog Wardrobe (1)
Update: 
- Bastokan Ring now shows up. 
 - The augments are not displaying in remarks, not sure if this is intended or not. 

### Mog Wardrobe (2)
Fixed: 
- Archer's Knife displays 2 times

### Mog Wardrobe (3)
Fixed:
- Field Boots

### Mog Wardrobe (4)
Fixed: 
- Harp

**Comment by TeoTwawki** [comment]
> The augments are not displaying in remarks, not sure if this is intended or not.

Will look into if I can make those display or not. They are not part of the usual item help text, I might not be able to detect if an item was augmented in the cached client file. Thanks for trying my test build for me.

**Comment by Xebeth** [comment]
Unless things have changed significantly since I developed this, item quantities are not store in the inventory cache files.
See https://x14.north-edge.com/lootbox#compact_listing

**Comment by TeoTwawki** [comment]
hi @Xebeth :wave: 

Correct about the quantities, they still aren't there. A lot of things are not sadly.

I forget the details atm, but what I found out was the "invisible items" were most often a result of coming after an item I had not dealt with properly in one of the slots - I found one item I could place is a given slot of inventor and any item after that would vanish in lootbox. Change the slot it is in and the point at which items stop showing changes. And on private servers at least, gaining new items or hitting sore can change the slot position (I think retail does this but did not check, as my account is no longer active). I haven't taken the time to fix it since finding that out yet though. I've been using an analysis tool to reduce the technical debt first which I have taken a big bite out of so far.
